integration_tests/test_aggregate_samples.py

```diff
@@ -0,0 +1,115 @@
+import os
+import time
+from datetime import datetime, timedelta
+
+import numpy as np
+import pandas as pd
+
+import datareservoirio as drio
+
+
+def wait_for_data_to_appear(func, delay=3):
```

Comment:
Is this 3 in seconds, does that not make the tests slow? For testing, ms is perhaps faster without slowing down tests?

Reply:
I assume this was the waiting for integration test.

Reply:
Yes, this is just to wait until the data appears in the integration test, but it takes around 5-7 seconds for the data to appear, so we might do 2 or 3 loops. Just thought 3 sec is a good balance of not spamming the API vs number of wait loops.